src/main/java/org/datadog/jenkins/plugins/datadog/DataDogJobProperty.java

```diff
@@ -0,0 +1,148 @@
+package org.datadog.jenkins.plugins.datadog;
+
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.model.Descriptor;
+import hudson.model.Job;
+import hudson.model.JobProperty;
+import hudson.model.JobPropertyDescriptor;
+import hudson.model.Run;
+import java.io.IOException;
+import java.util.logging.Logger;
+import net.sf.json.JSONObject;
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.StaplerRequest;
+
+/**
+ * Create a job property for use with DataDog plugin.
+ * @param <T>
+ */
+public class DataDogJobProperty<T extends Job<?,?>> extends JobProperty<T> {
+
+  private static final Logger LOGGER =  Logger.getLogger(DatadogBuildListener.class.getName());
+  private static final String DISPLAY_NAME = "DataDog Tagging";
+  private String tagging = null;  
+  private String tagFile = null;
+  
+  /**  
+   * This is a list of tagging targets to be submitted with the Build to DataDog.
+   * @return the tagging
+   */
+  public String getTagging() {
+    return tagging;
+  }
+
+  @DataBoundConstructor
+  public DataDogJobProperty( ) { }
+
+  @DataBoundSetter
+  public void setTagging(final String tagging) {
+    this.tagging = tagging;
+  }
+  
+  /**
+   * This method is called whenever the Job form is saved. We use the 'on' property 
+   * to determine if the controls are selected.
+   * 
+   * @param req - The request
+   * @param form - The submitted form
+   * @return The configured {@link JobProperty}
```

Comment:
`@return a {@link JobProperty} object representing the tagging added to the job`